pkg/monitoring/rules/recordingrules/vmi.go

```diff
@@ -23,6 +23,14 @@ import (
 )
 
 var vmiRecordingRules = []operatorrules.RecordingRule{
+	{
+		MetricsOpts: operatormetrics.MetricOpts{
+			Name: "kubevirt_vmi_phase_count",
+			Help: "Sum of VMIs per phase and node. `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`].",
+		},
+		MetricType: operatormetrics.GaugeType,
+		Expr:       intstr.FromString("sum by (node, phase, os, workload, flavor, instance_type, preference) (kubevirt_vmi_info)"),
+	},
```

Comment:
Why do we need this recording rule?
Can't `cnv:vmi_status_running:count` use the new metric directly?

Reply:
Its being used in other places and we want to keep backwards compatibility.